ddtrace/contrib/django/patch.py

```diff
@@ -331,26 +336,35 @@ def traced_get_response(django, pin, func, instance, args, kwargs):
         return func(*args, **kwargs)
 
     trace_utils.activate_distributed_headers(pin.tracer, int_config=config.django, request_headers=request.META)
+    request_headers = utils._get_request_headers(request)
+
+    with _asm_context.asm_request_context_manager(
+        request.META.get("REMOTE_ADDR"), request_headers, headers_case_sensitive=django.VERSION < (2, 2)
+    ):
+        with pin.tracer.trace(
+            "django.request",
+            resource=utils.REQUEST_DEFAULT_RESOURCE,
+            service=trace_utils.int_service(pin, config.django),
+            span_type=SpanTypes.WEB,
+        ) as span:
+            # set component tag equal to name of integration
+            span.set_tag_str("component", config.django.integration_name)
 
-    with pin.tracer.trace(
-        "django.request",
-        resource=utils.REQUEST_DEFAULT_RESOURCE,
-        service=trace_utils.int_service(pin, config.django),
-        span_type=SpanTypes.WEB,
-    ) as span:
-        # set component tag equal to name of integration
-        span.set_tag_str("component", config.django.integration_name)
+            utils._before_request_tags(pin, span, request)
+            span._metrics[SPAN_MEASURED_KEY] = 1
 
-        utils._before_request_tags(pin, span, request)
-        span._metrics[SPAN_MEASURED_KEY] = 1
+            response = None
 
-        response = None
-        try:
-            response = func(*args, **kwargs)
-            return response
-        finally:
-            # DEV: Always set these tags, this is where `span.resource` is set
-            utils._after_request_tags(pin, span, request, response)
+            try:
+                if config._appsec_enabled and _context.get_item("http.request.blocked", span=span):
+                    response = HttpResponseForbidden(
+                        appsec_utils._get_blocked_template(request_headers.get("Accept")), **kwargs
```

Comment:
I am not sure `**kwargs` should be here ?

This function wrapper is around [BaseHandler.get_response](https://github.com/django/django/blob/f3c89744cc801cc7d134bca9958c4a74aa76380f/django/core/handlers/base.py#L136) which it's only argument is a `request` object.

The function signature does not match that of an `HttpResponse` class, so I think we should avoid `**kwargs` here otherwise we are likely to have an issue.

```suggestion
                        appsec_utils._get_blocked_template(request_headers.get("Accept")),
```